app/controllers/games_controller.rb

```diff
@@ -19,8 +19,8 @@ def create
 	end
 
 	def update
-		render status: 400, json: {
-			text: 'Invalid Move'
+		render status: 200, json: {
+			text: 'Valid Move'
 		}
```

Comment:
This will be a conflict but nothing too difficult to fix

Reply:
So this is going to be obsolete on my next pr, I think, so don't worry about this too much at present.